opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -11,20 +12,42 @@ use std::{cell::RefCell, str, time::SystemTime};
 
 thread_local! { static EBW: RefCell<EventBuilder> = RefCell::new(EventBuilder::new());}
 
-/// UserEventsExporter is a log exporter that exports logs in EventHeader format to user_events tracepoint.
-pub(crate) struct UserEventsExporter {
-    provider: Mutex<Provider>,
-    name: String,
-    event_sets: Vec<Arc<EventSet>>,
-    cloud_role: Option<String>,
-    cloud_role_instance: Option<String>,
+/// Options for configuring the User Events Exporter.
+#[derive(Debug, Clone)]
+pub struct ExportOptions {
+    provider_name: String,
 }
 
-const EVENT_ID: &str = "event_id";
+impl ExportOptions {
+    /// Returns the provider name.
+    pub fn provider_name(&self) -> &str {
+        &self.provider_name
+    }
+
+    /// Builder for `EventsOptions`.
+    pub fn builder(provider_name: impl Into<String>) -> ExportOptionsBuilder {
+        ExportOptionsBuilder::new(provider_name)
+    }
+}
+
+/// Builder for `ExportOptions`.
+#[derive(Debug, Clone)]
```

Comment:
Do we want to implement `Clone`? It's probably okay but adds another public API.

```suggestion
#[derive(Debug)]
```